.github/workflows/python-integrate.yaml

```diff
@@ -70,10 +71,13 @@ jobs:
       - name: Buf Setup
         if: steps.pathChanges.outputs.sdkSchema == 'true' || steps.pathChanges.outputs.proto == 'true'
         uses: bufbuild/buf-setup-action@v1
+      - name: Buf Lint
+        if: steps.pathChanges.outputs.proto == 'true'
+        uses: bufbuild/buf-lint-action@v1
         with:
-          github_token: ${{ github.token }}
-
+          input: "proto"
       - name: Build Protobufs
+        if: steps.pathChanges.outputs.sdkSchema == 'true' || steps.pathChanges.outputs.proto == 'true'
```

Comment:
Good call with that. 

As I look, I believe the same `if` condition should be put for `Install SDK Schema` and `Run SDK Schema unit tests`